src/easyscore.ts

```diff
@@ -57,10 +57,11 @@ export class EasyScoreGrammar implements Grammar {
       expect: [this.DURATION, this.TYPE, this.DOTS, this.OPTS],
     };
   }
-  CHORDORNOTE(): Rule {
+  CHORDORNOTEORGHOST(): Rule {
```

Comment:
This doesn't feel right, the grammar doesn't change here, and we shouldn't have to special-case ghost notes.

Reply:
The change is the one you have commented below.

Reply:
Reverted